src/Microsoft.IdentityModel.LoggingExtensions/Microsoft.IdentityModel.LoggingExtensions.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.IdentityModel.LoggingExtensions</AssemblyName>
+    <Description>A package containing logging extensions to support writing Identity logs to Microsoft.Extensions.Logging.ILogger.</Description>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <PackageId>Microsoft.IdentityModel.LoggingExtensions</PackageId>
+    <PackageTags>.NET;Windows;Authentication;Identity;Extensions;Logging</PackageTags>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
As the assembly only targets netstandard2.0 I was unable to create a test project due to compatibility issues.
* xunit.running.visualstudio doesn't support netstandard2.0
* Adding additional targets in test projects break compatibility with LoggingExtensions assembly as it only supports netstandard2.0